<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>550</width>
    <height>510</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Object tracker AIO</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="statusText">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>290</y>
      <width>221</width>
      <height>91</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
      <weight>50</weight>
      <bold>false</bold>
     </font>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="text">
     <string>Awaiting Image Input</string>
    </property>
    <property name="scaledContents">
     <bool>false</bool>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
    <property name="wordWrap">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_IC">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>10</y>
      <width>221</width>
      <height>81</height>
     </rect>
    </property>
    <property name="text">
     <string>Initiate Camera</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_CC">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>100</y>
      <width>221</width>
      <height>81</height>
     </rect>
    </property>
    <property name="text">
     <string>Calibrate Camera</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_DLT">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>190</y>
      <width>221</width>
      <height>81</height>
     </rect>
    </property>
    <property name="text">
     <string>Draw Line Test</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_OT">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>10</y>
      <width>221</width>
      <height>71</height>
     </rect>
    </property>
    <property name="text">
     <string>Object Tracking</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_PP">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>90</y>
      <width>221</width>
      <height>71</height>
     </rect>
    </property>
    <property name="text">
     <string>Plot Path</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_RP">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>250</y>
      <width>221</width>
      <height>71</height>
     </rect>
    </property>
    <property name="text">
     <string>Reverse Proj</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_AP">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>330</y>
      <width>221</width>
      <height>71</height>
     </rect>
    </property>
    <property name="text">
     <string>Analysize Path</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_SFCSV">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>410</y>
      <width>481</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>Save path data to CSV</string>
    </property>
   </widget>
   <widget class="QLabel" name="savedRes">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>450</y>
      <width>481</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="alignment">
     <set>Qt::AlignHCenter|Qt::AlignTop</set>
    </property>
    <property name="wordWrap">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_SP">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>170</y>
      <width>221</width>
      <height>71</height>
     </rect>
    </property>
    <property name="text">
     <string>Smoothen Path</string>
    </property>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionClose">
   <property name="text">
    <string>Close</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
